Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM golang:1.13.8-buster
+
+RUN go get go.mongodb.org/mongo-driver/mongo
```

Comment:
Sugestão ou TODO: Adicionar um TODO para compilar o programa aqui. Dessa forma você não precisa nem ter Golang instalado na máquina. 

[Nessa página](https://www.cloudreach.com/en/resources/blog/containerize-this-how-to-build-golang-dockerfiles/) você encontra exemplos de imagens bem pequenas que compilam e rodam binários em Go.

Reply:
Sim. Acho que vai ser o ideal.
Nesse caso deixaremos disponível uma configuração para ambiente de desenvolvimento e outra para ambiente de execução?